sys/hacl/build.rs

```diff
@@ -107,9 +107,14 @@ fn create_bindings(platform: &Platform, home_dir: &Path) {
         // Allow function we want to have in
         .allowlist_function("Hacl_AEAD_Chacha20Poly1305_.*")
         .allowlist_function("Hacl_Curve25519.*")
+        .allowlist_function("Hacl_Hash_SHA1.*")
         .allowlist_function("Hacl_Hash_SHA2.*")
-        .allowlist_function("Hacl_Hash_.*")
-        .allowlist_function("Hacl_Blake2.*")
+        .allowlist_function("Hacl_Hash_SHA3.*")
+        .allowlist_function("Hacl_Hash_Blake2.*")
```

Comment:
I thought it would just document better what exactly we want and what we don't want. Also, this let me exclude `Hacl_Hash_SHA3_Scalar_sha3*`, `Hacl_Hash_SHA3_Scalar_shake128` and the like from the extraction (they weren't being used and I didn't want to add them in).